.eslintrc.cjs

```
@@ -0,0 +1,92 @@
+  ignorePatterns: [
+    'dist',
+    'node_modules',
+    'coverage',
+    '*.js',
+    '*.d.ts',
+    '.eslintrc.js',
+    '**/*.test.ts',
+    '**/*.test.tsx',
+    '**/*.spec.ts',
+    '**/*.spec.tsx',
+    'src/generated/**/*'
```

ESLint

[error] Insert `,`

(prettier/prettier)